storage/SystemInfo/CmdBtrfs.cc

```diff
@@ -182,9 +182,13 @@ namespace storage
 	    Entry entry;
 
 	    string::size_type pos1 = line.find("ID ");
+	    if (pos1 == string::npos)
+		ST_THROW(Exception("cound not find id"));
```

Comment:
(in all three places): 'could'

BTW, could that message be a bit more descriptive, like 'could not find id in btrfs output' or so?

Reply:
Sure.